modules/mergeImageWithTelemetry/mergeImageWithTelemetryWorker.py

```diff
@@ -0,0 +1,39 @@
+from modules.mergeImageWithTelemetry.mergeImageWithTelemetry import MergeImageWithTelemetry
+
+def pipelineMergeWorker(pause, exitRequest, imagePipelineIn, telemetryPipelineIn, pipelineOut):
+    print("start pipeline merge")
+    mergeImageWithTelemetry = MergeImageWithTelemetry()
+    curImage = None
+
+    while True: 
+        if not exitRequest.empty():
+            print("end pipeline merge")
+            return
+        
+        pause.acquire()
+        pause.release()
+
+        # put current elements of the telemetry pipeline into the queue
+        telemetryPipelineIn.acquire()
```

Comment:
A separate lock is supposed to be passed into the worker function. The queue/pipeline itself cannot be locked like this.